components/engine_traits/src/perf_context.rs

```diff
@@ -47,7 +48,8 @@ pub trait PerfContextExt {
 pub enum PerfContextKind {
     RaftstoreApply,
     RaftstoreStore,
-    GenericRead,
+    Storage(&'static str),
```

Comment:
The two names are a bit confusing, is the `Storage` type for non-coprocessor read prcoessing?